tests/jobserver/api/test_releases.py

```diff
@@ -224,10 +280,10 @@ def test_release_index_api_logged_in_no_permission():
 
 @pytest.mark.django_db
 def test_release_index_api_have_permission():
-    release = ReleaseFactory()
+    release = ReleaseFactory(ReleaseUploadsFactory(["file.txt"]))
     user = UserFactory()
     client = APIClient()
-    index_url = f"/api/v2/releases/{release.id}"
+    index_url = f"/api/v2/releases/release/{release.id}"
```

Comment:
Same here.

Reply:
Yeah, these are left over from when I had all the permissions checks in the same test, rather than split into individual tests.